app/models/category.rb

```diff
@@ -2,6 +2,7 @@ class Category < ApplicationRecord
   belongs_to :user
 
   validates :amount, :title, presence: true
-  validates :amount, numericality: {greater_than_or_equal_to: 0}
+  validates :amount, presence: true, format: {with: /\A\d+(?:\.\d{0,2})?\z/},
+                     numericality: {greater_than_or_equal_to: 0, less_than: 1_000_000_00}
```

Comment:
`100_000_000`

Reply:
@AleksSenkou I looked [here](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/NumericLiterals) and used syntax for cents.
`1_000_000_00` means `1 000 000.00` not 100 000 000